packages/node/src/lib/http-client.ts

```diff
@@ -43,16 +55,15 @@ export interface HTTPClientRequest {
   /**
    * HTTP method to be used for the request. This will always be a 'POST' request.
    **/
-  method: 'POST'
+  method: 'POST' | (string & {})
```

Comment:
The ` | (string & {})` lets you pass any string in, but also still allows editors to suggest the known strings.

Benefit is the editor will only show you the method we support, but we can add additional ones in the future as needed without it breaking anyones builds.

Reply:
Total nit: but couple downsides off the top of my head 

- someone would  have to remember to update that interface years down the line as the compiler won't remind them  if we they ever add another method, since this is basically kinda just a glorified doc string now. 
- if there is a new method and someone us currently using a custom http client that only supports POST, when they update to our package, the compiler will not warn them that this is a breaking change

Reply:
Ok will change it to just POST. We can do a major bump if we need to support other types in the future.